Sources/WebAuthn/Ceremonies/Authentication/PublicKeyCredentialRequestOptions.swift

```diff
@@ -114,6 +140,14 @@ public struct PublicKeyCredentialDescriptor: Equatable, Encodable, Sendable {
         try container.encode(id.base64URLEncodedString(), forKey: .id)
         try container.encodeIfPresent(transports, forKey: .transports)
     }
+    
+    public init(from decoder: any Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        let type = try container.decode(CredentialType.self,forKey: .type)
+        let id = try container.decodeBytesFromURLEncodedBase64( forKey: .id)
+        let transports = try container.decodeIfPresent([AuthenticatorTransport].self,forKey:.transports) ?? []
+        self.init(type: type, id:id, transports: transports)
```

Comment:
Formatting, and let's use the same pattern as elsewhere for consistency:
```suggestion
        let container = try decoder.container(keyedBy: CodingKeys.self)

        self.type = try container.decode(CredentialType.self, forKey: .type)
        self.id = try container.decodeBytesFromURLEncodedBase64(forKey: .id)
        self.transports = try container.decodeIfPresent([AuthenticatorTransport].self, forKey: .transports) ?? []
```